Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -35,4 +28,56 @@ function GetVssConnection
     return $vssConnection
 }
 
+#
+# The issues, as reported by SonarQube, need to be transformed to a simpler structure that the PostComments module can consume
+# 
+function GetCommentsFromIssues
+{
+    param ([Array]$issues)
```

Comment:
Because PowerShell does not natively support types I decided to work with general purpose objects like Array and PSObject. Ideally the comments should be transformed to a data structure exposed by the Discussion TFS APIs, but this isn't the case today.